src/api/correlation/propagation.rs

```diff
@@ -215,6 +215,19 @@ mod tests {
                         "key4=123.567",
                 ],
             ),
+        // "values of array types"
+            (
+                vec![
+                    KeyValue::new("key1", Value::Array(vec![Value::Bool(true), Value::Bool(false)])),
+                    KeyValue::new("key2", Value::Array(vec![Value::String("val1".to_string()), Value::String("val2".to_string())])),
+                    KeyValue::new("key3", Value::Array(vec![Value::I64(123), Value::I64(456)])),
+                ],
+                vec![
+                        "key1=[true%2Cfalse]",
+                        "key2=[val1%2Cval2]",
```

Comment:
Think the spec wants these to be json encoded values, so strings should probably be quoted here even though it complicates things a little.

Reply:
Oops, sorry about that. Fixed.